kpops/components/base_components/kafka_connector.py

```diff
@@ -148,7 +160,9 @@ def deploy(self, dry_run: bool) -> None:
 
     @override
     def destroy(self, dry_run: bool) -> None:
-        self.handlers.connector_handler.destroy_connector(self.name, dry_run=dry_run)
+        self.handlers.connector_handler.destroy_connector(
+            self.connector_name, dry_run=dry_run
```

Comment:
why a new property instead of simply `self.full_name`?

Reply:
I didn't have in mind that `self.full_name` exists :D